src/utils/index.ts

```diff
@@ -37,7 +37,7 @@ export const isEmptyList = <T,>(obj: Array<T> | T): boolean =>
  *
  */
 export const isEmptyObject = <T,>(obj: Object | T): boolean =>
-    !obj ||
-    typeof obj !== 'object' ||
-    Array.isArray(obj) ||
+    typeof obj === 'object' &&
```

Comment:
These changes need to be reverted, as expected behaviour is `isEmptyObject` must return true for non-object type values.